src/javascript/app_2/Stores/Modules/SmartChart/smart-chart-store.js

```diff
@@ -167,10 +182,16 @@ export default class SmartChartStore extends BaseStore {
         this.should_export_layout = false;
         this.should_import_layout = true;
         this.should_clear_chart   = false;
-        this.trade_chart_layout.isDone = action(() => this.trade_chart_layout = {});
+        this.trade_chart_layout.isDone = action(() => this.trade_chart_layout = null);
```

Comment:
Could you add `this.should_import_layout = false` inside isDone too? I forgot to add it before.